docs/Generators/SiFive-Generators.rst

```diff
@@ -0,0 +1,45 @@
+SiFive Generators
+==================
+
+Chipyard includes several open-source generators developed and maintained by `SiFive <https://www.sifive.com/>`__. 
+These are currently organized within two submodules named ``sifive-blocks`` and ``sifive-cache``.
+
+L2 Cache
```

Comment:
I feel like a better name is last level cache generator, as it does not support any caches closer to DRAM than it.